packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -406,6 +410,12 @@ class CupertinoDatePicker extends StatefulWidget {
   /// Whether to to show day of week alongside day. Defaults to false.
   final bool showDayOfWeek;
 
+  /// The uniform height of all children.
+  ///
+  /// All children will be given the [BoxConstraints] to match this exact
+  /// height. Must not be null and must be positive.
```

Comment:
Can you add a sentence describing that it will default to a preset value? Maybe something like "Will default to a value that matches the default iOS date picker wheel."